src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -114,6 +114,31 @@ def load_arguments(self, _):
         c.argument('subscription_required', arg_group='API', arg_type=get_three_state_flag(), help='If true, the API requires a subscription key on requests.')
         c.argument('tags', tags_type)
 
+    with self.argument_context('apim nv create') as c:
+        c.argument('service_name', options_list=['--service-name'], help='The name of the API Management service instance.')
+        c.argument('named_value_id', help='The id of the API Management Named Value.')
```

Comment:
Which kinds of id? GUID?

Reply:
Not necessarily. Could be a normal name, a GUID or some random string.